Dockerfile

```diff
@@ -8,7 +8,10 @@ RUN apt-get install -y -q --no-install-recommends \
 
 WORKDIR /go/src/github.com/docker/cli
 
-RUN git clone https://github.com/docker/cli.git .
+RUN git clone https://github.com/ulyssessouza/cli . && git checkout 29d30ec1a05e8a285007240cc0f85864067e7344
+# FIXME(ulyssessouza): Go back to the line below when PRs https://github.com/docker/cli/pull/1710 and https://github.com/docker/cli/pull/1690 hits the cli
```

Comment:
No need to change, but it might actually end up being https://github.com/docker/cli/pull/1718 in preference to https://github.com/docker/cli/pull/1710

Reply:
I made this change in my rebase